src/frontend/apps/desk/src/pages/ConfigLayout.tsx

```diff
@@ -21,17 +16,11 @@ export default function RootLayout({
   const { theme } = useCunninghamTheme();
 
   return (
-    <html lang="en">
-      <body suppressHydrationWarning={process.env.NODE_ENV === 'development'}>
-        <QueryClientProvider client={queryClient}>
-          <ReactQueryDevtools />
-          <CunninghamProvider theme={theme}>
-            <Auth>
-              <InnerLayout>{children}</InnerLayout>
-            </Auth>
-          </CunninghamProvider>
-        </QueryClientProvider>
-      </body>
-    </html>
+    <QueryClientProvider client={queryClient}>
+      <ReactQueryDevtools />
+      <CunninghamProvider theme={theme}>
+        <Auth>{children}</Auth>
+      </CunninghamProvider>
+    </QueryClientProvider>
```

Comment:
As it's not related to HTML layout, I would rename this component, `AppProvider`? Idk, what do you think?